relay-server/src/services/processor/profile_chunk.rs

```diff
@@ -17,8 +17,13 @@ use {
 /// Removes profile chunks from the envelope if the feature is not enabled.
 pub fn filter<G>(state: &mut ProcessEnvelopeState<G>) {
     let continuous_profiling_enabled = state.project_info.has_feature(Feature::ContinuousProfiling);
```

Comment:
I wonder, can we instead of adding a new feature just use the existing `Feature::ContinuousProfiling` and either remove or add it in the feature handler?

Reply:
> I wonder, can we instead of adding a new feature just use the existing `Feature::ContinuousProfiling` and either remove or add it in the feature handler?

@Dav1dde @jjbayer I agree it makes sense and indeed yesterday in the team weekly we discussed doing just that, but at the end I was asked not to mix the 2 things and just create a new one. So here we are.